### ISSUE TITLE
Btags not working on latest commit?

### ISSUE THREAD

**Issue by shyun3** [ibhagwan/fzf-lua#372]
### Info

- Operating System: WSL Ubuntu 20.04
- Shell: Bash
- Terminal: Windows Terminal
- `nvim --version`: 0.6.0
- `fzf --version`: 0.27.0

- [x] The issue is reproducible with `minimal_init.lua`

### Description 

I updated `fzf-lua` to commit c815531 only to find that `:FzfLua btags` does not seem to work anymore.

Here's a minimal setup:
```bash
mkdir -p ~/btags-test
cd ~/btags-test
echo "int main() {}" > test.cpp
ctags -R
nvim test.cpp
```

In Neovim, entering `:FzfLua btags` opens the pop-up window with no entries. `:FzfLua tags` works fine with the sole entry being the `main()` function. Is this a bug or with the new tags changes, am I using `btags` incorrectly now?

**Comment by ibhagwan** [comment]
I forgot to add global `rg_opts` to `btags` and it messes up the command, this will be fixed soon along with the other issue you opened.

In the meantime you can use `btags({ rg_opts = '' })`.

**Comment by ibhagwan** [comment]
https://github.com/ibhagwan/fzf-lua/commit/1d9331532825623701dae0265143673f8d84185f - should work now, I'll have the fix for #371 soon, just making sure other special regex characters are handled as well.
